apis/fluent-en_GB/atrium-api-fluent-en_GB-js/build.gradle

```diff
@@ -1,4 +1,4 @@
-description = 'A fluent assertion function API in en_GB with a focus on code completion for the JS platform.'
+description = 'A fluent assertion function API in en_GB with a focus on code completion as js module.'
```

Comment:
fine with me as well but could you change the other places then too please? I prefer to be consistent. You could search & replace with regex enabled:
search: `for the (.*) platform` 
replace with: `as $1 module`
It's enough to search in *.gradle files

Reply:
i added as a js test was failing with a no description error message

Reply:
not sure what you mean, this description is used by gradle, I don't see why it should interfere with a js test. Can you elaborate please

Reply:
I do not recall exactly but it was when I was trying to run a grade build in the multiplatform directory and it was saying 'description can't be null' or something along those lines. When I added it, it stopped throwing that error

Reply:
OK, in this case revert your change please. This modifies an existing description and this cannot be the cause.

Reply:
reverted change and
`You need to define project.description for publishing (empty or blank is considered to be undefined)
`

Reply:
I guess you deleted it. Just revert to the previous description

Reply:
it was empty

Reply:
i'll go ahead and copy this
https://github.com/robstoll/atrium/blob/master/apis/fluent-en_GB/atrium-api-fluent-en_GB-js/build.gradle
not sure why when i checked out master it came empty for me